Source/SIMPLib/DataContainers/AttributeMatrixProxy.cpp

```diff
@@ -285,3 +278,63 @@ void AttributeMatrixProxy::updatePath(DataArrayPath::RenameType renamePath)
     dataArrays.insert(newPath.getDataArrayName(), daProxy);
   }
 }
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+DataArrayProxy& AttributeMatrixProxy::getDataArrayProxy(const QString& name)
+{
+  for(QMap<QString, DataArrayProxy>::iterator iter = dataArrays.begin(); iter != dataArrays.end(); ++iter) // DataArray Level
+  {
+    DataArrayProxy& dataArray = iter.value();
+    if(dataArray.name.compare(name) == 0)
+    {
+      return dataArray;
+    }
+  }
+
+  DataArrayProxy proxy;
+  proxy.name = name;
+  dataArrays.insert(proxy.name, proxy);
+  return dataArrays[name];
+}
+
+QMap<QString, DataArrayProxy> AttributeMatrixProxy::getDataArrays()
+{
+	return dataArrays;
```

Comment:
getDataArrays through setFlag all have extra indentation and are missing the comment block above their definitions unlike other methods.